src/js/components/Select/StyledSelect.js

```diff
@@ -40,15 +40,15 @@ export const HiddenInput = styled.input`
 export const SelectOption = styled(Button)`
   ${(props) => props.selected && props.textComponent && selectedStyle}
   // applies theme.global.hover.background to the active
-  // option for mouse and keyboard interactions
+    // option for mouse and keyboard interactions
```

Comment:
Seems like some unnecessary space was added here, potentially due to different local linting/prettier set up?